docs/core/project-sdk/msbuild-props.md

```diff
@@ -416,6 +416,37 @@ Restoring a referenced package installs all of its direct dependencies and all t
 </PropertyGroup>
 ```
 
+## COM properties and items
+
+- [EnableComHosting](#enablecomhosting)
+- [EnableDynamicLoading](#enabledynamicloading)
+
+For more information, see [Expose .NET components to COM](../native-interop/expose-components-to-com.md).
+
+### EnableComHosting
+
+The `EnableComHosting` property indicates that an assembly provides a COM server. Setting the `EnableComHosting` to `true` also implies that [EnableDynamicLoading](#enabledynamicloading) is `true`.
+
+```xml
+<PropertyGroup>
+  <EnableComHosting>True</EnableComHosting>
+</PropertyGroup>
+```
+
+### EnableDynamicLoading
+
+The `EnableDynamicLoading` property indicates that an assembly is a [COM library](/windows/win32/com/the-component-object-model). Setting this property to `true` has the following effects:
```

Comment:
`EnableDynamicLoading` is not specifically for COM. It is intended to be for any dynamically loaded component - for example, a (non-COM) library that can be used from a [native host](https://docs.microsoft.com/dotnet/core/tutorials/netcore-hosting#create-a-host-using-nethosth-and-hostfxrh).